docs/contributing/testing.md

```diff
@@ -65,27 +65,36 @@ You should test that your contribution works:
 
 ## 5. Run the automated tests
 
+Automated testing helps ensure that our code is as free of bugs as it can be. It also lets us know immediately if a change we make breaks any other part of the code.
+
 In your project folder, run `npm test` to run the automated tests, including linting.
 
 If a test fails, you should check your code for any errors, then update any tests you need to.
```

Comment:
Are they only updating the tests that fail?

Reply:
I think there probably is a scenario where someone may need to update tests that *didn't* fail. They might spot some tests which should have failed, but didn't - they should also update those to make sure the tests are specific enough that they will catch code changes in future.

Reply:
Well of we get any feedback that people are not sure what we mean then we can revisit.